ext/tokenizer/tokenizer_data_gen.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+$infile = 'Zend/zend_language_parser.h';
+$outfile = 'ext/tokenizer/tokenizer_data.c';
+
+if (!file_exists($infile)) {
+    fwrite(STDERR, <<<ERROR
+    $infile is missing.
+    
+    Please, generate the PHP parser files by scripts/dev/genfiles
+    or by running the ./configure build step.
+    ERROR);
+    exit(1);
+}
+
+$result = '';
+
+$result .= <<<CODE
+/*
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Author: Johannes Schlueter <johannes@php.net>                        |
```

Comment:
Doesn't look right. :)

Reply:
Well, a copyright notice for *generated* files looks fishy, but an author makes certainly no sense. However, I'm fine with sticking with the status quo here.